packages/api/src/services/actions/lsp.ts

```diff
@@ -47,9 +47,10 @@ export function Handlers(config: Config, appState: Dependencies): Actions {
       addresses = addresses ? lodash.castArray(addresses) : [];
       return queries.sumTvl(addresses, currency);
     },
-    // lsp does not support tvm at this time, but we want a function here for consistency with emp interface
-    async tvm() {
-      return "0";
+    async tvm(addresses: string[] = [], currency: CurrencySymbol = "usd") {
```

Comment:
this is what gets exposed in the api. it follows the pattern and logic established for emps. you can basically provide an array of addresses, or an empty array which will sum all known tvms.  This also ends up being used on the global channel in order to sum tvm across lsps and emps